packages/theme/src/core/searchBox/index.ts

```diff
@@ -0,0 +1,31 @@
+import colors from '../colors';
+
+const SearchBox = {
+    sizes: {
+        S: {
+            height: '4rem',
+            iconPosition: '0.7rem'
+        },
+        M: {
+            height: '4.8rem',
+            iconPosition: '1.1rem'
+        }
+    },
+    bgColor: colors.white,
```

Comment:
Should we wrap these colors like these colros: {default: {}, active: {}, hover: {}} ??